publish_rate: 100 # Hz, at which everything is republished from node

# Controller gains
p_gain_pos: 50
d_gain_pos: 2.5

p_gain_ori: 0.01
d_gain_ori: 0.0005